script/DeployV2.s.sol

```diff
@@ -39,15 +40,27 @@ contract DeployV2 is Script {
 
         vm.startBroadcast();
 
-        PrincipalToken pt = new PrincipalToken(dtPrediction);
-        DelegateToken dt = new DelegateToken(address(registry), ptPrediction, baseURI, deployer);
-        market = new WrapOfferer(seaport15, dtPrediction);
-
-        console2.log("ptAddress:", address(pt));
-        console2.log("dtAddress:", address(dt));
-
-        require(address(pt) == ptPrediction, "wrong sim");
-        require(address(dt) == dtPrediction, "wrong sim");
+        PrincipalToken principalToken = new PrincipalToken(dtPrediction);
+        DelegateTokenStructs.DelegateTokenParameters memory delegateTokenParameters = DelegateTokenStructs.DelegateTokenParameters({
+            delegateRegistry: address(registry),
+            principalToken: ptPrediction,
+            baseURI: baseURI,
+            initialMetadataOwner: deployer
+        });
+        DelegateToken delegateToken = new DelegateToken(delegateTokenParameters);
+        OffererStructs.Parameters memory createOffererParameters = OffererStructs.Parameters({
+            seaport: seaport15,
+            seaportConduit: seaportConduit,
```

Comment:
Dependency on specific conduits is interesting, previously we didn't need to specify this. Does this limit us in any way?

Reply:
So, the intention is to verify that the caller of `transferFrom` in `CreateOfferer` is the conduit the buyer / seller signer specified in their order. We can add that into the hash instead of restricting `CreateOfferer` to a specific conduit, would achieve the same goal without that inflexibility of conduit upgrades / custom conduit usage.